velox/common/encode/Coding.h

```diff
@@ -273,6 +273,10 @@ class ZigZag {
     return (static_cast<uint64_t>(val) << 1) ^ (val >> 63);
   }
 
+  static __uint128_t encodeInt128(__int128_t val) {
```

Comment:
Can we have a test for this? Thanks!

Reply:
Added `TEST_F(ZigZagTest, hugeInt)`. Thanks.